flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["i386","arm","aarch64"]
```

Comment:
```suggestion
  "skip-arches": ["aarch64"]
```

Reply:
Our emulator does not support any 32-bit CPU architectures so they have to be skipped